docs/en/latest/plugins/limit-conn.md

```diff
@@ -32,17 +32,30 @@ The `limit-conn` Plugin limits the number of concurrent requests to your service
+| key_type                 | string  | False    | "var"       | ["var", "var_combination"]        | Type of user specified key to use.                                                                                                                                                                                                                                                                                                                                                                        |
+| key                      | string  | True     |             |                                   | User specified key to base the request limiting on. If the `key_type` attribute is set to `"var"`, the key will be treated as a name of variable, like `remote_addr` or `consumer_name`. If the `key_type` is set to `"var_combination"`, the key will be a combination of variables, like `$remote_addr $consumer_name`. If the value of the key is empty, `remote_addr` will be set as the default key. |
+| rejected_code            | string  | False    | 503         | [200,...,599]                     | HTTP status code returned when the requests exceeding the threshold are rejected.                                                                                                                                                                                                                                                                                                                         |
+| rejected_msg             | string  | False    |             | non-empty                         | Body of the response returned when the requests exceeding the threshold are rejected.                                                                                                                                                                                                                                                                                                                     |
+| allow_degradation        | boolean | False    | false       |                                   | When set to `true` enables Plugin degradation when the Plugin is temporarily unavailable and allows requests to continue.                                                                                                                                                                                                                                                                                 |
+| policy             | string  | False    | local | local, redis, redis-cluster | counter type to choose local, redis or redis-cluster                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
the description could be a bit more descriptive for `policy` like [limit-count's](https://apisix.apache.org/docs/apisix/plugins/limit-count) doc?